fe1-web/src/features/linked-organizations/screens/LinkedOrganizationsScreen.tsx

```diff
@@ -0,0 +1,432 @@
+import { ListItem, FAB } from '@rneui/themed';
+import React, { useState } from 'react';
+import { Text, View, Modal, StyleSheet, ViewStyle, ScrollView } from 'react-native';
+import { TouchableWithoutFeedback } from 'react-native-gesture-handler';
+import { useToast } from 'react-native-toast-notifications';
+
+import { DatePicker, Input, PoPButton, PoPIcon, PoPTextButton, QRCode } from 'core/components';
+import ModalHeader from 'core/components/ModalHeader';
+import PoPTouchableOpacity from 'core/components/PoPTouchableOpacity';
+import QrCodeScanner, { QrCodeScannerUIElementContainer } from 'core/components/QrCodeScanner';
+import QrCodeScanOverlay from 'core/components/QrCodeScanOverlay';
+import ScreenWrapper from 'core/components/ScreenWrapper';
+import { Hash, Timestamp } from 'core/objects';
+import { List, ModalStyles, Spacing, Typography, Color } from 'core/styles';
+import { accent, contrast } from 'core/styles/color';
+import { container } from 'core/styles/list';
+import { FOUR_SECONDS } from 'resources/const';
+import STRINGS from 'resources/strings';
+
+import { LinkedOrganizationsHooks } from '../hooks';
+import { Challenge } from '../objects/Challenge';
+import { Organization } from '../objects/Organization';
+
+const initialOrganizations: Organization[] = [];
+
+const styles = StyleSheet.create({
+  qrCode: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    opacity: 0.5,
+    top: '25%',
+    bottom: '50%',
+  } as ViewStyle,
+  scannerTextItems: {
+    top: '35%',
+  } as ViewStyle,
+  enterButton: {
+    ...QrCodeScannerUIElementContainer,
+    borderColor: Color.blue,
+    borderWidth: Spacing.x025,
+    alignSelf: 'center',
+    marginBottom: Spacing.contentSpacing,
+  } as ViewStyle,
+  flex1: {
```

Comment:
If possible, I would try to give the classes meaningful names

Reply:
This was marked as resolved, I guess you don't agree? 😂

Reply:
Sorry I overlooked that!